src/authentication/components/SecuredRoute.tsx

```diff
@@ -55,8 +56,20 @@ const SecuredRoute: FunctionComponent<SecuredRouteProps & RouteComponentProps> =
 			render={props => {
 				// Already logged in
 				if (loginState && loginState.message === LoginMessage.LOGGED_IN) {
-					const Component = component;
-					return <Component />;
+					if (isProfileComplete()) {
+						const Component = component;
+						return <Component />;
```

Comment:
Iets dat ik nog had opgemerkt, alle route component zouden geen `withRouter` nodig moeten hebben maar bij ons wel. Dat komt omdat je hier die props (location, history en match) vergeet terug op de Component te zetten.
Mss een TODO zetten dat we dat nog fixen.

Reply:
fixed and tested with search component
na deze merge kunnen we dan geleidelijk alle withRouters verwijderen